pkg/servers/servers_test.go

```diff
@@ -27,6 +29,17 @@ var ready func(rw http.ResponseWriter, req *http.Request)
 var mux *http.ServeMux
 var httpClient, httpsClient *http.Client
 
+func awaitConnection(address string, numberOfRetries int, retryInterval time.Duration) error {
```

Comment:
can be replaced with eventually
for example https://github.com/openshift/assisted-service/blob/master/subsystem/kubeapi_test.go#L117